docker.env

```diff
@@ -9,4 +9,4 @@ API_SERVICES=["auth"]
 AUTH_API_SERVICE={"name": "Authentication", "endpoint": "/auth", "port": 3000, "defaultVersion": 1, "availableVersions": [1]}
 QUERY_API_SERVICE={"name": "Query", "endpoint": "/query", "port": 4000, "defaultVersion": 1, "availableVersions": [1]}
 MEMCACHE_SERVICE={"keyPrefix": "docker", "cacheHost": "memcached:11211"}
-KEYCLOAK_SERVICE={"host": "0.0.0.0", "clientId": "", "clientSecret": ""}
+KEYCLOAK_SERVICE={"host": "http://keycloak:8080/auth/realms/clin", "clientId": "clin-proxy-api", "clientSecret": "01b99f28-1331-4fec-903b-c2e8043cec77"}
```

Comment:
@latch2112 clientSecret in docker?

Reply:
yup... only for dev and QA

Reply:
to change manually in prod... even though keycloak won't be accessible